oneview_redfish_toolkit/api/computer_system_collection.py

```diff
@@ -23,28 +23,30 @@ class ComputerSystemCollection(RedfishJsonValidator):
     """Creates a Computer System Collection Redfish dict
 
         Populates self.redfish with some hardcoded ComputerSystemCollection
-        values and with the response of Oneview with all server hardware.
+        values and with the response of Oneview with all servers profiled
+        applied.
     """
 
     SCHEMA_NAME = 'ComputerSystemCollection'
 
-    def __init__(self, server_hardware):
+    def __init__(self, server_profiles):
```

Comment:
Seeing the blueprint code I believe that server_profiles name is bad in this case because this list is a server_hardware list, in fact, right? So, we can named as server_hardware.